sites/friday/src/manage-resources/resource-init.md

```diff
@@ -13,42 +13,111 @@ keywords:
+and [activating](#environment-activation) an environment, or [restoring a backup](#backup-restoration).
+
+## Default resources
+
+{{% vendor/name %}} allocates the following default resources to every container when deployed for the first time:
 
 | Resource type               | Amount |
 | --------------------------- | ----------- |
 | CPU                         | 0.5 |
 | RAM                         | Depends on the [container profile](/manage-resources/adjust-resources.md#advanced-container-profiles). |
 | Disk size (only applicable if the app or service requires a disk)                   | 512 MB |
 
-If you don't want to use those default resources, you can define a different [resource initialization strategy](/manage-resources/resource-init.md#define-a-resource-initialization-strategy).
-You can also [adjust those resources](/manage-resources/adjust-resources.md) after your project or new container has been deployed.
+If you don't want to use these default resources, you can specify a [resource initialization strategy](/manage-resources/resource-init.md#specify-a-resource-initialization-strategy).
+You can also [adjust resources](/manage-resources/adjust-resources.md) after your project or new container has been deployed.
+
+{{% note %}}
 
 For information on costs related to resource usage, see the [{{% vendor/name %}} pricing page](https://upsun.com/pricing/).
 Note that you can [keep an eye on those costs](/manage-resources/resource-billing.md) in the Console.
 
-## Define a resource initialization strategy
+{{% /note %}}
 
-You can define which strategy {{% vendor/name %}} uses to allocate resources
-when you first deploy your project or add a new container.
-The following strategies are available:
+## Specify a resource initialization strategy
+
+{{% vendor/name %}} provides the following resource initialization strategies:
 
 | Strategy | Description |
 | ---------| ----------- |
-| `default`  | Initializes the new containers using the [{{% vendor/name %}} default resources](/manage-resources/resource-init.md). |
-| `manual`   | With this strategy, the first deployment fails and you need to configure resources manually through [the Console](/manage-resources/adjust-resources.md), or using `resources:set` in the CLI. |
-| `minimum`  | Initializes the new containers using the [{{% vendor/name %}} minimum resources](#minimum-resources). |
-| `parent`   | Initializes the new containers using the same resources as the parent environment.</br>If there is no parent environment, or if the container doesn't already exist on the parent, the `default` strategy applies instead. |
+| `default`  | Initializes new containers using the [{{% vendor/name %}} default resources](#default-resources).</br>This strategy applies when you first deploy your project (or new container) unless you explicitly specify a different strategy, or allocate resources manually via the `resources:set` CLI command.  |
```

Comment:
There is no anchor on this page with an id of `default-resources` so Hugo is interpreting this as an empty `href`